lib/muser_ctx.c

```diff
@@ -1226,7 +1226,7 @@ lm_ctx_try_attach(lm_ctx_t *lm_ctx)
 }
 
 lm_ctx_t *lm_create_ctx(lm_trans_t trans, const char *path, int flags,
```

Comment:
No need to fix now, but we should be starting functions at the start of the line, like:

lm_ctx_t *
lm_create_ctx...

Reply:
Sure